content/applications/inventory_and_mrp/manufacturing/workflows/work_center_time_off.rst

```diff
@@ -1,57 +1,53 @@
-============================================
-Make work centers unavailable using Time Off
-============================================
+====================
+Work center time off
+====================
+
+.. |MO| replace:: :abbr:`MO (manufacturing order)`
 
 In Odoo, *work centers* are used to carry out manufacturing operations at specific locations.
 However, if a work center cannot be used for some reason, work orders begin to pile up at the work
 center until it is operational again.
 
-As a result, it is necessary to make the work center unavailable in Odoo so that the platform routes
-new work orders to alternative work centers that are operational. Using Odoo *Time Off*, it is
+As a result, it is necessary to make the work center unavailable in Odoo so new work orders are
+routed to alternative work centers that are operational. Using Odoo's **Time Off** app, it is
 possible to designate a work center as being unavailable for a set period of time. Doing so ensures
 that manufacturing operations can continue until the impacted work center is available again.
 
 Configuration
 =============
 
 Before a work center can be designated as unavailable, the Odoo platform must be properly
-configured. First, it is necessary to enable :ref:`developer mode <developer-mode>`. This allows the
-:guilabel:`Time Off` smart button to appear on each work center's :guilabel:`Working Hours` pop-up
-window.
-
-Enable developer mode by going to :menuselection:`Settings`, scrolling to the bottom of the page,
-and clicking :guilabel:`Activate the developer mode` under the :guilabel:`Developer Tools` heading.
+configured. First, it is necessary to enable :doc:`developer mode
+<../../../general/developer_mode>`. This allows the :guilabel:`Time Off` smart button to appear on
+each work center's :guilabel:`Working Hours` page.
 
-.. image:: work_center_time_off/developer-mode-button.png
-   :align: center
-   :alt: The "Activate the developer mode" button.
+Enable developer mode by going to the :menuselection:`Settings` app, scrolling to the bottom of the
+page, and clicking :guilabel:`Activate the developer mode`, under the :guilabel:`Developer Tools`
+heading.
 
-Next, install the *Time Off* app. This is the app used for assigning time off to all resources
-within Odoo, including employees and work centers. Navigate to :menuselection:`Apps`, then type
-`Time Off` in the :guilabel:`Search...` bar. The card for the :guilabel:`Time Off` module should be
-the only one that appears on the page. Click the green :guilabel:`Install` button on the card to
-install the app.
+Next, install the **Time Off** app. This is the app used for managing time off for all resources
+within Odoo, including employees and work centers.
 
-.. image:: work_center_time_off/time-off-install-card.png
-   :align: center
-   :alt: The Time Off module installation card.
+To do so, navigate to the :menuselection:`Apps` app, then type `Time Off` in the search bar. The
```

Comment:
```suggestion
To do so, navigate to the :menuselection:`Apps` app, then search `Time Off` in the search bar. The
```
opting for this slight change of phrase because, if a reader were to only follow along with what they read, they *potentially* wouldn't know to hit 'Enter', or actually **search** for this term. They might even think the results auto-populate. Either way, I think this suggested change would circumnavigate any potential issues here.